lib/rubygems/specification.rb

```diff
@@ -2462,6 +2462,7 @@ def to_ruby
       next if handled.include? attr_name
       current_value = self.send(attr_name)
       if current_value != default_value(attr_name) || self.class.required_attribute?(attr_name)
+        require "openssl"
```

Comment:
We were not requiring "openssl" before https://github.com/rubygems/rubygems/pull/2782 🤔

Reply:
No, but after #2782 we're using the `OpenSSL` constant, so we need to require `openssl`. At least that's what the error is telling us.

Reply:
Ahh you are right, I missed that.